packages/services/skills/nightshift/SKILL.md

```
@@ -0,0 +1,41 @@
+description: Check Night Shift status, view recent PRs, or manually trigger a Night Shift run.
+---
+
+# Night Shift
+
+Manage the 4am autonomous coding service.
+
+**Arguments**: $ARGUMENTS — action (status | trigger | history)
+
+## Actions
+
+### status (default)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)